File: AumentoSalarial.java
import java.util.Scanner;

public class AumentoSalarial {
    public static void main(String[] args) {
        Scanner leitor = new Scanner(System.in);
        String nome;
        float sal;
        float reaj;
        float nsal;

        System.out.print("Nome do funcionário: ");
        nome = leitor.next();
        System.out.print("Salário R$: ");
        sal = leitor.nextFloat();

        System.out.print("Reajuste (%): ");
        reaj = leitor.nextFloat();

        nsal = sal + ( sal * reaj /100 );

        System.out.println( nome + " ganhava R$ " + sal + " e depois de ganhar " + reaj + "% de aumento vai passar a ganhar R$ " + nsal);


    }
}

File: AumentoSalarial.kt
fun main() {
    print("Nome do Funcionário: ")
    var nome:String = readLine()!!
    print("Salário R$: ")
    var salario:Float = readLine()!!.toFloat()
    print("Reajuste %: ")
    var reaj:Float = readLine()!!.toFloat()
    println("$nome que ganhava R$ $salario de salário e depois do reajuste de $reaj % passará a ganhar ${salario+(salario*reaj/100)}.")


    //sal + ( sal * reaj /100 )

}